src/gwt/www/index.htm

```diff
@@ -17,16 +17,17 @@
 <!-- standards mode -->
 <html>
   <head>
-	
+
     <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
     <meta name="gwt:property" content="compiler.stackMode=#compiler_stack_mode#"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
```

Comment:
Is this intended to apply to RStudio Server too?

Reply:
Yes -- we also want to prevent users of Safari / Chrome on macOS from getting into this zoomed-in state as well.

Reply:
I think there will be some number of users who are on smaller devices (e.g. mini ipads) who are going to be willing to do short RStudio Server sessions, and will want to double tap on the part of the UI they want to interact with. It's not a pleasant experience, but it makes using RStudio merely annoying instead of impossible. I'm okay with what you have now as a first volley but we should revisit when we fix https://github.com/rstudio/rstudio/issues/2186.